packages/text-editor/src/components/TextEditor.svelte

```diff
@@ -48,7 +48,11 @@
       dispatch('content', content)
     }
   }
-
+  export function setContent (newContent: string): void {
+    content = newContent
+    editor.commands.setContent(content)
```

Comment:
We will lose Undo history, please update only if content is different.

Reply:
Fixed, but we have one bug - TicTac's setContent() cuts spaces from content at the beginning and at the end.

Reply:
This bug also fixed